src/backend/src/utils/reimbursement-requests.utils.ts

```diff
@@ -15,12 +15,38 @@ export interface ReimbursementProductCreateArgs {
   wbsElementId: number;
 }
 
+export interface ReimbursementReceiptCreateArgs {
+  name: string;
+  googleFileId: string;
+}
+
 export interface ReimbursementProductCreateArgs {
   name: string;
   cost: number;
   wbsElementId: number;
 }
 
+/**
+ * This function removes any deleted receipts and adds any new receipts
+ * @param receipts the new list of receipts to compare against the old ones
+ * @param currentReceipts the current list of receipts on the request that's being edited
+ */
+export const updateReceiptPictures = async (receipts: ReimbursementReceiptCreateArgs[], currentReceipts: Receipt[]) => {
+  if (currentReceipts.length > 0) {
```

Comment:
make this if (currentReceipts.length === 0) return;

Reply:
also add currentReceipts.length === receipts.length right?

Reply:
no cause you could be adding a new one and remove an old one and end up at the same number